types/koa/test/index.ts

```diff
@@ -37,7 +37,7 @@ app.use<{}, UserContext>(async ctx => {
     ctx.user = {};
 });
 
-app.use((ctx, next) => {
+app.use((ctx: Koa.Context, next) => {
```

Comment:
This change is not related to the subject of this PR and should be done in a separate PR.

Reply:
Same as above.